bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -907,21 +935,34 @@ public void stop() {
 }
 
 @Override
-public boolean setText(String html, boolean trusted) {
-	char[] data = new char[html.length() + 1];
-	html.getChars(0, html.length(), data, 0);
-	return webView.NavigateToString(data) == COM.S_OK;
+public URI getBaseUri() {
+	return BASE_URI;
+}
+
+private void writeToDefaultPathDOM() {
+	if(html != null && URI.create(getUrl()).equals(BASE_URI)) {
```

Comment:
I guess that this should use `isLocationForCustomText()`.

Reply:
Yes, changed it.